lemminx-maven/src/main/java/org/eclipse/lemminx/extensions/maven/MavenProjectCache.java

```diff
@@ -235,4 +274,12 @@ public PlexusContainer getPlexusContainer() {
 	public Collection<MavenProject> getProjects() {
 		return projectCache.values();
 	}
+	
+	private DOMDocument createDOMDocument(URI uri) throws IOException, URISyntaxException {
```

Comment:
Replace this method with https://github.com/eclipse/lemminx/blob/d27272d14db97cba7f22a22d7a7c7bc104d28d14/org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/utils/DOMUtils.java#L128

Reply:
ok